databuilder/models/table_metadata.py

```diff
@@ -165,10 +179,12 @@ def get_description_id(self) -> str:
         else:
             return "_" + self.source + "_description"
 
-    def __repr__(self) -> str:
-        return f'DescriptionMetadata({self.source!r}, {self.text!r})'
+    def get_description_default_key(self, start_key: Optional[str]) -> Optional[str]:
```

Comment:
should this return optional?

Reply:
start_key may be none.
https://github.com/amundsen-io/amundsendatabuilder/pull/461/files#diff-1db7e313c9a410ad2ab2ba0622407c8b7f131b2d395090e403a1af95a9b04dacR154